src/context/LocationProvider.tsx

```diff
@@ -43,9 +44,11 @@ export const LocationProvider = ({
     }
   }, []);
 
-  // This probably were u wanna make location call more than once big dawg
   useEffect(() => {
     fetchLocation();
+
+    // Set up interval to fetch location every 5 seconds
+    setInterval(fetchLocation, 5000);
```

Comment:
save the location fetch cooldown as a variable in `constant.ts` so this variable can be configured in one place